services/github/github-hacktoberfest.service.js

```diff
@@ -0,0 +1,194 @@
+    <ul>
+      <li>
+        The number of suggested issues. By default this will count open
+        issues with the <strong>hacktoberfest</strong> label, however you
+        can pick a different label (e.g.
+        <code>?suggestion_label=good%20first%20issue</code>).
+      </li>
+      <li>
+        The number of pull requests opened in October. This excludes any
+        PR with the <strong>invalid</strong> label.
+      </li>
+      <li>The number of days left of October.</li>
+    </ul>
+
+  </p>
+
+  ${githubDocumentation}
+`
+
+const schema = Joi.object({
+  data: Joi.object({
+    repository: Joi.object({
+      issues: Joi.object({
+        totalCount: nonNegativeInteger,
+      }).required(),
+    }).required(),
+    search: Joi.object({
+      issueCount: nonNegativeInteger,
+    }).required(),
+  }).required(),
+}).required()
+
+const queryParamSchema = Joi.object({
+  suggestion_label: Joi.string(),
+}).required()
+
+module.exports = class GithubHacktoberfestCombinedStatus extends GithubAuthV4Service {
+  static get category() {
+    return 'issue-tracking'
+  }
+
+  static get route() {
+    return {
+      base: 'github/hacktoberfest/2019',
+      pattern: ':user/:repo',
+      queryParamSchema,
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'GitHub Hacktoberfest combined status',
+        namedParams: {
+          user: 'snyk',
+          repo: 'snyk',
+        },
+        staticPreview: this.render({
+          suggestedIssueCount: 12,
+          contributionCount: 8,
+          daysLeft: 15,
+        }),
+        documentation,
+      },
+      {
+        title:
+          'GitHub Hacktoberfest combined status (suggestion label override)',
+        namedParams: {
+          user: 'tmrowco',
+          repo: 'tmrowapp-contrib',
+        },
+        queryParams: {
+          suggestion_label: 'help wanted',
+          contribution_label: undefined,
+        },
+        staticPreview: this.render({
+          suggestedIssueCount: 12,
+          contributionCount: 8,
+          daysLeft: 15,
+        }),
+        documentation,
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'hacktoberfest',
+      color: 'orange',
+    }
+  }
+
+  static render({ suggestedIssueCount, contributionCount, daysLeft }) {
+    if (daysLeft === undefined) {
+      daysLeft = moment('2019-11-01').diff(moment(), 'days')
+    }
+
+    const message =
+      [
+        suggestedIssueCount ? `${metric(suggestedIssueCount)} open issues` : '',
+        contributionCount ? `${metric(contributionCount)} PRs` : '',
+        daysLeft > 0 ? `${daysLeft} day${daysLeft > 1 ? 's' : ''} left` : '',
+      ]
+        .filter(Boolean)
+        .join(', ') || 'is done!'
+
+    return { message }
+  }
+
+  async fetch({ user, repo, suggestionLabel = 'hacktoberfest' }) {
+    const isValidOctoberPR = [
+      `repo:${user}/${repo}`,
+      'is:pr',
+      'created:>2019-10-01',
```

Comment:
Do you want to also add a cap for 1st Nov? (It looks like the days left was capped)